lib/cron.dart

```diff
@@ -47,6 +47,24 @@ class Schedule {
   /// The weekdays a Task should be started.
   final List<int>? weekdays;
 
+  /// The second a Task last run.
+  int? lastSecond = -1;
```

Comment:
I think these fields should be private (`_lastSecond`) and if you initialize them, they can be `int` instead of `int?`. e.g. `int _lastSecond = -1;`.